web/src/components/base/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, ReactNode } from 'react'
+import styled from 'styled-components'
+
+const StyledButton = styled.button<{ disabled: boolean }>`
+  background-color: transparent;
+  cursor: ${props => (props.disabled ? 'default' : 'pointer')};
+  pointer-events: ${props => (props.disabled ? 'none' : 'default')};
+  padding: 0;
+  border: none;
+  text-align: start;
+`
+
+type ButtonProps = {
+  onClick: () => void
+  children?: ReactNode
+  ariaLabel: string
+  disabled?: boolean
+  tabIndex?: number
+  className?: string
+}
+
+const Button = ({ onClick, children, ariaLabel, tabIndex, className, disabled = false }: ButtonProps): ReactElement => (
+  <StyledButton
+    type='button'
+    onClick={onClick}
+    disabled={disabled}
+    aria-label={ariaLabel}
```

Comment:
Please also add aria role button:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Why? This doesn't make sense for a `<button>` tag.
> The button role is for clickable elements that trigger a response when activated by the user. Adding role="button" tells the screen reader the element is a button, but provides no button functionality. Use [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) or [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) with type="button" instead.